pages/legal/imprint.js

```diff
@@ -234,80 +239,76 @@ export default withData(({url}) => {
           <Highlight>Werden ungeduldig erwartet</Highlight><br />
 
       Adelina Gashi, Ausbildung «Unternehmerischer Journalismus»<br />
-      Elia Blülle, Ausbildung «Unternehmerischer Journalismus»<br />
           <A href='/~0c536be1-69a5-4dd3-adb4-9533cb3f7a9d'>
         Isabelle Schwab
-      </A>
+          </A>
       , Ausbildung «Unternehmerischer Journalismus»<br />
+      Katrin Moser, Produktion<br />
           <A href='/~909e78a3-27c9-40c0-8b95-7814f46f441b'>
-        Michael Kuratli
-      </A>
-      , Ausbildung «Unternehmerischer Journalismus»<br />
-          <A href='/~3b1cee6c-cc48-431a-ade7-60f739c66298'>
-        Solmaz Khorsand
-      </A>
-      , Reporterin<br />
+      Michael Kuratli
+          </A>
+    , Ausbildung «Unternehmerischer Journalismus»<br />
     Urs Bruderer, Bundeshaus-Autor
 
-    </Item>
+        </Item>
 
       </List>
 
       <H2>Gestaltung</H2>
 
       <List>
         <Item>
-          Bodara GmbH (Logo, Schrift Republik Serif, Gestaltung Screen und Print)<br />
+        Bodara GmbH (Logo, Schrift Republik Serif, Gestaltung Screen und Print)<br />
           <A href='/~bfbab748-e3a1-4478-a298-fbb4fd79e9cc'>
-            Bahar Büyükkavir
+          Bahar Büyükkavir
           </A>
-          , Gestalterin<br />
+        , Gestalterin<br />
           <A href='/~92360ee3-b142-44a0-a044-65dbc36d5827'>
-            Dominique Schmitz
+          Dominique Schmitz
           </A>
-          , Gestalter<br />
+        , Gestalter<br />
           <A href='/~a1bd424d-0388-4966-a430-5b4551ed90e4'>
-            Mark Philip Ruoss
+      Mark Philip Ruoss
```

Comment:
Indentation goes a bit wild here, try to keep as it was before.

Rule of thumb:
Opening tag followed by new line: indent 2 spaces on the new line.